index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dan's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section id="skeleton">
+      <header>
+        <h1 id="welcome">Welcome</h1>
```

Comment:
- [ REQUIRED] AS the last reviewer mentioned Please wrap the logo 'Welcome' in a tag,
@dd-obua if you look at most of the websites you will notice that pressing the logo will send you to the home page of this website we want you to do the same here so your project looks like professional websites